Osdocs 9783 take4 (#76321)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

[discrete]
[id="ocp-4-16-sha-haproxy-support-removed"]
=== SHA-1 certificates no longer supported for use with HAProxy

> SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].

Reviewer: I might leave the {product-title} in there to be clear. I like the addition of the version number.
```suggestion
SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in {product-title} 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Author: Sure thing!

> SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].

Author: oh my, I should have caught that one!

[id="ocp-4-16-sha-haproxy-support-removed"]

> === SHA1 certificates no longer supported for use with HAProxy

Reviewer 2: Supposedly, SHA-1 is the proper spelling for SHA1. I didn't know this until recently. For example, https://en.wikipedia.org/wiki/SHA-1. 
 
```suggestion
=== SHA-1 certificates no longer supported for use with HAProxy
```

Author: ty

> SHA1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].

Reviewer 2: ```suggestion
SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Author: ty
